html/global_attributes.json

```
@@ -695,7 +695,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": "≤13.1"
```

Matches HTMLElement API data

```suggestion
              "version_added": "12.1"
```